terraform/cyhy_public_acl_rules.tf

```diff
@@ -0,0 +1,91 @@
+# Allow ingress from anywhere via the Nessus UI and ssh ports
```

Comment:
I don't think we need to allow ingress from anywhere via the Nessus UI port.  At this point the only way to access the Nessus UI is via ssh port forwarding.

Reply:
Agreed!